code/modules/research/designs.dm

```diff
@@ -604,7 +588,45 @@
 	build_path = /obj/item/weapon/gun/energy/cutter/plasma
 	sort_string = "TAEAC"
 
-
+/datum/design/item/weapon/divet
+	name = "Winchester NK Divet pistol"
+	id = "divet"
+	req_tech = list(TECH_COMBAT = 1, TECH_MATERIAL = 3)
+	materials = list(MATERIAL_STEEL = 10000, MATERIAL_SILVER = 4000)
+	build_path = /obj/item/weapon/gun/projectile/divet
+	sort_string = "TACEA"
+
+/datum/design/item/weapon/divetslug
+	name = "divet magazine"
+	id = "divetslug"
+	req_tech = list(TECH_COMBAT = 2, TECH_MATERIAL = 3)
```

Comment:
Tech_material to 2. Sorry if I've not been clear.

Magazines are almost universally easier to obtain than their actual weapon counterpart.